_posts/2021-02-09-january-rubygems-update.md

```diff
@@ -0,0 +1,37 @@
+---
+title: January 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in January.
+
+## RubyGems News
+
+In January, we released new versions of bundler `v2.2.7` and `v2.2.8`, and rubygems `v3.2.7` and `v3.2.8`, the most important change being to fix resolver edge cases and speed up bundler - [#4277](https://github.com/rubygems/rubygems/pull/4277) this change makes bundler faster and the resolver more correct.
+
+We also made the following improvements and fixes:
+
+- looked up information for a CI related issue where the the `--silent` flag did not work with `gem update --system` - [#2907](https://github.com/rubygems/rubygems/issues/2907#issuecomment-756957235) and that was resolved on [#3789](https://github.com/rubygems/rubygems/pull/3789)
+- worked on RFC for Bundler Version Locking -[#29](https://github.com/rubygems/rfcs/pull/29).
+- got pending releases out and kept working on more performance issues in bundler.
+
+This month, [RubyGems](https://github.com/rubygems/rubygems) gained [172 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-01-02%7D...master@%7B2021-01-31%7D), contributed by 12 authors. There were 53323 additions and 1646 deletions across 2565 files.
+
+## RubyGems.org News
+
+This month in RubyGems.org, we improved versions endpoint response time which makes SQL query on `compact_index` versions up to **1400x times** faster by adding index on `indexed` and `yanked_at` columns - [#2615](https://github.com/rubygems/rubygems.org/pull/2615).
+
+In additon to that, we made the following improvement and fixes:
+
+- debugged delay in versions endpoint update on `gem push` and set surrogate key on versions to reduce fastly cache expiry - [#2612](https://github.com/rubygems/rubygems.org/pull/2612), [#2614](https://github.com/rubygems/rubygems.org/pull/2614)
+- worked on pre-update changes for **rails 6.1** and updated and deployed **rails 6.1** update [#2607](https://github.com/rubygems/rubygems.org/pull/2607), [#2597](https://github.com/rubygems/rubygems.org/pull/2597), [#2598](https://github.com/rubygems/rubygems.org/pull/2598)
+- created fastly support tickets for dedicated IPs and incorrect status code on matching If-None-Match.
+- worked on updating rubygems-terraform root files to sync with current state.
```

Comment:
Thanks to iMacTia, we added a new MFA level `UI and gem signin` https://github.com/rubygems/rubygems.org/pull/2601 Find out more about this at https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels